.changeset/three-chairs-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Update error log formatting
```

Comment:
We like to make these changesets a paragraph or two, like what would go in a blog post.